gpytorch/lazy/lazy_tensor.py

```diff
@@ -1924,6 +1960,16 @@ def transpose(self, dim1, dim2):
 
         return res
 
+    def type(self, dtype):
```

Comment:
I suppose at this point we could also implement `to` if we wanted to, since now we support `.cuda`, `.cpu`, and the dtype casts.

Reply:
Yeah, i could probably set that up in another PR. Currently, the `.to(...)` method has a bit different signature than the pytorch one so it'll require a bit of work.